forge-gui/res/draft/rankings.txt

```diff
@@ -38491,7 +38491,7 @@
 #103|False Summoning|C|PO2
 #104|Mind Rot|C|PO2
 #105|Dakmor Scorpion|C|PO2
-#106|Deja Vu|C|PO2
+#106|Déjà Vu|C|PO2
```

Comment:
I'm not sure this file is going to be imported this way and if its not, its going to be hard to maintain. Maybe we can keep this one with the normalized names?

Reply:
i don't know where the ranking file is built from

i need to debug to see how the ranking file is parsed to see if it would support normalized names

Reply:
I'm in the middle of splitting these up into individual files. I'd prefer if it wouldn't change right now.